File: src/fibonacci.cpp
// A program that calculates the first 10 numbers in the Fibonacci Sequence
// By: Michael Rutherford

#include <iostream>

int main() {
    int a = 0;
    int b = 1;
    int c = 0;

    std::cout << a << std::endl;
    std::cout << b << std::endl;

    for (int i = 0; i < 10; i++) {
        c = b + a;
        std::cout << c << std::endl;
        a = b;
        b = c;
    }

    return 0;
}

File: src/fibonacci.py
"""
A program that calculates the first 10 numbers in the Fibonacci Sequence
By: Michael Rutherford
"""

def main():
    a = 0
    b = 1
    c = 0

    print(a)
    print(b)

    for _ in range(10):
        c = b + a
        print(c)
        a = b
        b = c

if __name__ == "__main__":
    main()
